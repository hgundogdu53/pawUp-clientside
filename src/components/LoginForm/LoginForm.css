* {
    box-sizing: border-box;
}

.flex-container .login_form {
    text-transform: none;
}
.demo-credentials {
    display: block;
    margin-top: 15%;
    text-align: center;
    text-transform: none;
    border: 3px solid;
    padding: 5px 20px;

}
