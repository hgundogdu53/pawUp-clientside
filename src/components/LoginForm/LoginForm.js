
import React from 'react';
import './LoginForm.css';

function LoginForm(props) {
    return (
        <section className='flex-container'>
            <h2 className='login-title'>Log In</h2>
            <form className='login_form' onSubmit={props.loginUser}>
                <section className='login_form-container'>
                    <div>
                        <label htmlFor='login_email'>Email:</label>
                        <input type='email' name='login_email' id='login_email' />
                    </div>
                    <div>
                        <label htmlFor='login_password'>Password:</label>
                        <input type='password' name='password' id='login_password' />
                    </div>
                    <button type='submit' className='sub-btn'>Log in</button>
                    {props.error ? <h4>Incorrect Email or Password</h4> : <></>}
                </section>
                <div className='demo-credentials'>
                    <h3>Demo Credentials</h3>
                    <h4>e-mail: hgundogdu53@yahoo.com</h4>
                    <h4>Password: 123456Hg!</h4>
                </div>
            </form>

        </section>
    )
}

export default LoginForm;