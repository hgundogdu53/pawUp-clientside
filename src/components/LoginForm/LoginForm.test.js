import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LoginForm from './LoginForm';

describe(`LoginForm component`, () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<LoginForm />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});