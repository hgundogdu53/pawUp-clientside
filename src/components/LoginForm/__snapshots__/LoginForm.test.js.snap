// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm component renders without crashing 1`] = `
<section
  className="flex-container"
>
  <h2
    className="login-title"
  >
    Log In
  </h2>
  <form
    className="login_form"
  >
    <section
      className="login_form-container"
    >
      <div>
        <label
          htmlFor="login_email"
        >
          Email:
        </label>
        <input
          id="login_email"
          name="login_email"
          type="email"
        />
      </div>
      <div>
        <label
          htmlFor="login_password"
        >
          Password:
        </label>
        <input
          id="login_password"
          name="password"
          type="password"
        />
      </div>
      <button
        className="sub-btn"
        type="submit"
      >
        Log in
      </button>
    </section>
  </form>
</section>
`;
